moveit_ros/planning_interface/planning_scene_interface/src/planning_scene_interface.cpp

```diff
@@ -210,6 +214,19 @@ class PlanningSceneInterface::PlanningSceneInterfaceImpl
     return result;
   }
 
+  bool applyPlanningScene(const moveit_msgs::PlanningScene &planning_scene) const
```

Comment:
Yes, another neglected specifier that was shadowed by the `mutable` above.
I adjusted it. Let's see whether it passes now.